### ISSUE TITLE
Bring back the tab menu in other parcels: Information mode.

### ISSUE THREAD

**Issue by Benjythebee** [cryptovoxels/issues#104]
**Is your feature request related to a problem? Please describe.**
Removing the tab menu in other people's parcel removes a significant interaction with others and their parcels.

**Describe the solution you'd like**
The tab menu should work in other's parcel but instead of showing the 'edit mode', it should show basic information:
- Name of parcel
- Parcel description as supplied by the owner.
- Owner and contributors (with a link to their profile, i.e. cryptovoxels.com/Edmond for example)
- A list of NFTs if any

Note that I am saying "NFTs" and not Features in general because showing all features to the publix is just not a good idea.

**Additional context**
This menu could in the future be used for other items such as social interactions, eg:
- Thumb up / down the parcel,
- Number of thumbs up ...

**Comment by mmckegg** [comment]
This is somewhat provided by clicking on the address in the toolbar. 

What if when you clicked on the address, it had a drop down that gave you the information (instead of forcing you to leave the world to see the parcel info page -- we could still have a link to that from the drop down).

Forgive my quick and dirty mockup:

<img width="675" alt="Screen Shot 2020-06-24 at 5 37 41 PM" src="https://user-images.githubusercontent.com/66834/85505033-775e0000-b641-11ea-9488-8385f6d0f107.png">

**Comment by Benjythebee** [comment]
Yes that would be amazing as a start! It avoids people leaving the world. best method for player retention.
Don't forget a button to go to the parcel's page
